src/config/dynamic-theme-fixes.config

```diff
@@ -17956,6 +17956,26 @@ pre {
 
 ================================
 
+pathfinderwiki.com
+
+CSS
+body {
+    background-color: rgb(10,10,10);
+    background-image: none;
+}
+.mw-page-container-inner {
+    background-color: rgb(30,30,10);
+    background-image: none;
+}
+.fw-alignment-on {
+    --darkreader-inline-bgcolor: royalblue !important;
+}
+.fw-alignment-on a {
+    color: white !important;
```

Comment:
Again we don't want to force white
Change to:
`color: ${black} !important;`

BTW As we still be in touch during preview. Where exactly on the page this class is located? Can you share some screen and/or URL?